ooyala_player/ooyala_player.py

```diff
@@ -118,6 +119,19 @@ def _retrieve_transcript(self):
             pass
         return transcript
 
+    def _get_download_url(self):
+        """
+        Fetches the video's download URL from the backlot API.
+        """
+        client = OoyalaApi(self.api_key, self.secret_key)
+        path = 'assets/{content_id}/streams'.format(content_id=self.content_id)
```

Comment:
This doesn't seem right? The API docs [say](http://help.ooyala.com/video-platform/api/asset_video.html) "The term streams in this context means the internal-to-Ooyala processes that are started by the system to transcode your videos. Note: You cannot control or manipulate these streams; they are controlled by the system. You can only view their names, not change them."

Per the description of the task, we need to try using the http://help.ooyala.com/video-platform/api/syndications.html API instead.